packages/manager/modules/web-paas/src/components/additional-option/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,8 @@
+{
+  "web_paas_offer_additional_option_title_staging_environment": "Ajouter des environnements de test",
+  "web_paas_offer_additional_option_title_storage": "Ajouter du stockage (en Go)",
+  "web_paas_offer_additional_option_title_user_license": "Ajouter des licences utilisateur",
+  "web_paas_offer_additional_option_price_desc_staging_environment": "/mois",
+  "web_paas_offer_additional_option_price_desc_storage": "/environnement/mois",
+  "web_paas_offer_additional_option_price_desc_user_license": "/mois"
```

Comment:
duplicate value '/mois'

Reply:
Values are duplicated but we need these keys as it is a combination of a constant key and value fetched from the API result.